multicluster/apis/multicluster/v1alpha1/gateway_types.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GatewayInfo includes basic information of Gateway Node
+type GatewayInfo struct {
+	// The Gateway IP of the Gateway Node. It might be assigned by user manually
```

Comment:
Suggest to remove this comment. It does not provide any useful information.

Reply:
removed